framework/test/jest/unit/specs/modules/chain/synchronizer/synchronizer.spec.js

```diff
@@ -0,0 +1,376 @@
+/*
+ * Copyright © 2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+
+'use strict';
+
+const {
+	Synchronizer,
+} = require('../../../../../../../src/modules/chain/synchronizer/synchronizer');
+
+const blocksVerify = require('../../../../../../../src/modules/chain/blocks/verify');
+
+const {
+	Block: blockFixture,
+} = require('../../../../../../mocha/fixtures/blocks');
+
+jest.mock('../../../../../../../src/modules/chain/blocks/verify');
+
+const maxPayloadLength = 10000;
+const maxTransactionsPerBlock = 25;
+
+const storageMock = {
+	entities: {
+		Block: {
+			getLastBlock: jest.fn(),
+		},
+	},
+};
+
+const loggerMock = {
+	info: jest.fn(),
+};
+
+const syncParameters = {
+	storage: storageMock,
+	logger: loggerMock,
+	exceptions: {},
+	blockReward: {},
+	maxTransactionsPerBlock,
+	maxPayloadLength,
+};
+
+describe('synchronizer', () => {
+	afterEach(async () => {
+		jest.clearAllMocks();
+	});
+
+	describe('Synchronizer', () => {
+		let sync;
+		let sync1;
```

Comment:
Perhaps we could rewrite this variable names to `oneSynchronizer` and `anotherSynchronizer`